lib/superstore/adapters/jsonb_adapter.rb

```diff
@@ -0,0 +1,210 @@
+gem 'pg'
+require 'pg'
+
+module Superstore
+  module Adapters
+    class JsonbAdapter < AbstractAdapter
+      class QueryBuilder
+        def initialize(adapter, scope)
+          @adapter  = adapter
+          @scope    = scope
+        end
+
+        def to_query
+          [
+            "SELECT #{select_string} FROM #{@scope.klass.table_name}",
+            where_string,
+            order_string,
+            limit_string
+          ].delete_if(&:blank?) * ' '
```

Comment:
it isn't really a big deal, and probably not worth the rewrite.  I was just noticing that sometimes strings are created using `query = "SELECT #{select_string} ..."` and other times use use `Array#*`.  I'm assuming `Array#*` was done because it is faster, but if that's the case, why not consistently use it everywhere. On the other hand, if you like the clarity of string interpolation, why not write 

```
"SELECT #{select_string} FROM #{@scope.klass.table_name} #{where_string} #{order_string}  #{limit_string}
```